js/rendererjs/statusbar.js

```diff
@@ -15,9 +15,9 @@ const currentEstimatedHeight = () => {
 }
 
 // estimatedProgress returns the estimated sync progress given the current
-// blockheight, as a number from 0 -> 100
+// blockheight, as a number from 0 -> 99
 const estimatedProgress = (currentHeight) =>
-	currentHeight / currentEstimatedHeight() * 100
+	currentHeight / currentEstimatedHeight() * 99
```

Comment:
unfortunately I think this won't quite do it, because `currentEstimatedHeight()` is just that -- an estimate. It's possible for currentHeight to exceed `currentEstimatedHeight()`, which would push this into 100% again. So we need an explicit branch for `if > 99 then 99`